CorsixTH/Src/lua_sdl.h

```diff
@@ -40,6 +40,8 @@ SOFTWARE.
 // SDL_USEREVENT_SOUND_OVER - informs script of a played sound finishing.
 #define SDL_USEREVENT_SOUND_OVER (SDL_USEREVENT + 4)
 
+#define TICKS_MS_TO_USERTICKS_CONVERSION 30
```

Comment:
`constexpr usertick_period_ms = 30;` is a better name.

Reply:
Should we drop the Macro ?

Reply:
Yes. http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es31-dont-use-macros-for-constants-or-functions